crypto/fipsmodule/evp/digestsign.c

```diff
@@ -78,8 +79,19 @@ DEFINE_LOCAL_DATA(struct evp_md_pctx_ops, EVP_MD_pctx_ops) {
 }
 
 static int uses_prehash(EVP_MD_CTX *ctx, enum evp_sign_verify_t op) {
-  return (op == evp_sign) ? (ctx->pctx->pmeth->sign != NULL)
-                          : (ctx->pctx->pmeth->verify != NULL);
+  // Pre-hash modes in ML-DSA differ from other signing algorithms, so we
```

Comment:
```suggestion
  // Pre-hash mode of ML-DSA that uses an external mu calculation differs from other signing algorithms, so we
```

Reply:
added in [ba1856b](https://github.com/aws/aws-lc/pull/2113/commits/ba1856b920504afa5a4fb37ddf5ff80f70093b34)